scripts/lang/fr/__init__.py

```diff
@@ -855,6 +864,17 @@ def last_template_handler(template: Tuple[str, ...], locale: str) -> str:
         phrase = century(parts, "siècle") if parts else "Siècle à préciser"
         return term(phrase)
 
+    # Handle the {{fr-verbe-flexion}} template
+    if tpl == "fr-verbe-flexion":
+        phrase = ""
+        data = extract_keywords_from(parts)
+        if data["1"]:
+            phrase = data["1"]
+        else:
+            tag = [p for p in parts if "=" not in p]
```

Comment:
`parts` cannot contains strings with "=", those are removed by `extract_keywords_from()`.
You could simply reduce the whole thing to:
```python
data = extract_keywords_from(parts)
return data.get("1", parts[0])
```